### ISSUE TITLE
ergoCub 1.3 S/N:002 – Head Monitor Not Working

### ISSUE THREAD

**Issue by andrearosasco** [robotology/icub-tech-support#1808]
### Robot Name 🤖

ergoCub 1.1 S/N:002

### Request/Failure description

The ergocub head monitor does not show anything

### Detailed context

The ergoCub head monitor does not show anything as if it is disconnected from `ergocub-head`. 
After restarting the robot the monitor still does not work

**Comment by maggia80** [comment]
@andrearosasco next time could you log the `dmsg` output?

**Comment by andrearosasco** [comment]
Yes but for a while we will not be using `ergoCub 1.1 S/N:002` . I'm gonna tag @S-Dafarra since I guess AMI will be preparing the Washington demo on it

**Comment by S-Dafarra** [comment]
One interesting thing that @davidelasagna was thinking about is that, since now the monitor and the head start together, maybe at the beginning the monitor is not able to draw enough power, blocking the head from detecting it.

**Comment by maggia80** [comment]
@S-Dafarra yes, there could be an issue related to the power supply. I guess it could be more related to the USB HUB. We might connect the power of the display directly to the power source of the Jetson, instead of passing through the USB HUB.

**Comment by AntonioConsilvio** [comment]
Hello!

I have been monitoring this issue, and in the last month it does not seem to have reoccurred.

However, I would like to ask you for feedback if you have noticed this issue in the last period.

cc @S-Dafarra @GiulioRomualdi @andrearosasco

**Comment by S-Dafarra** [comment]
I haven't noticed this either!

**Comment by AntonioConsilvio** [comment]
Hi! The monitor seems working properly now! Closing!
